README.md

```diff
@@ -75,8 +76,10 @@ Usage:
 Available Commands:
   completion  Generate the autocompletion script for the specified shell
   describe    Describe all validation results in a Kubernetes cluster
+  docs        Display information about supported validator plugins
   help        Help about any command
-  install     Install validator & configure validator plugin(s)
+  install     Install validator & validator plugin(s)
+  rules       Configure & apply, or directly evaluate rules validator plugin rules.
```

Comment:
nit: `evaluate rules validator plugin rules.` should be `evaluate ~~rules~~ validator plugin rules.

Reply:
yep, good catch. i fixed it in code after updating the README 🤦